pygeoweaver/utils.py

```diff
@@ -146,3 +146,28 @@ def copy_files(source_folder, destination_folder):
             )
             os.makedirs(os.path.dirname(destination_file), exist_ok=True)
             shutil.copy2(source_file, destination_file)
+
+
+def create_table(data, max_length=100):
```

Comment:
Not sure this is a good idea. Also, a table is not `impressive` improvement comparing to all the other libraries. What we need is a widget that have `button` and dropdown, selection, text editors, things like that.